bindings/rust/standard/bench/src/harness/mod.rs

```diff
@@ -381,6 +406,15 @@ mod tests {
 
                         assert!(conn_pair.negotiated_tls13());
                         assert_eq!(cipher_suite, conn_pair.get_negotiated_cipher_suite());
+
+                        // read in "application data" handshake messages. 
+                        // "Client Finished" in the case of MutualAuth,
+                        // "NewSessionTicket" in the case of resumption
+                        let err = conn_pair.client_mut().recv(&mut[0]).unwrap_err();
```

Comment:
This should never be ClientFinished for s2n-tls, and I'd be very surprised if it could be ClientFinished for other TLS implementations. You can get an error bc the ClientFinished was bad here, but not the ClientFinished itself.